video_xblock/static/js/player_context_menu.js

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
I know our js file names are a bit messy, but we need to stick to something. Let it be dashes rather than underscores.

So can you please rename it to `player-context-menu.js`